mRemoteV1/Resources/Language/Language.de.resx

```diff
@@ -2617,4 +2618,39 @@ Wählen Sie einen Pfad für die mRemoteNG-Protokolldatei</value>
   <data name="PlaceSearchBarAboveConnectionTree" xml:space="preserve">
     <value>Suchleiste über den Verbindungen anzeigen</value>
   </data>
+  <data name="strCreateEmptyPanelOnStartUp" xml:space="preserve">
+    <value>Erstelle ein leeres Panel, wenn mRemoteNG gestartet wird</value>
+  </data>
+  <data name="strReleaseChannel" xml:space="preserve">
+    <value>Update-Kanal</value>
+  </data>
+  <data name="strReleaseChannelExplanation" xml:space="preserve">
+    <value>Stable enthält nur finale Versionen.
+Beta beinhaltet Betas &amp; Release Candidates.
+Development umfasst Alphas, Betas und Release Candidates.</value>
+  </data>
+  <data name="strTrackActiveConnectionInConnectionTree" xml:space="preserve">
+    <value>Aktive Verbindung im Verbindungsbaum verfolgen</value>
+  </data>
+  <data name="strEncryptionKeyDerivationIterations" xml:space="preserve">
+    <value>Iterationen der Schlüsselableitungsfunktion</value>
+  </data>
+  <data name="strLoadBalanceInfoUseUtf8" xml:space="preserve">
+    <value>UTF8-Kodierung für RDP Eigenschaft "Load Balance Info" verwenden</value>
+  </data>
+  <data name="Credentials" xml:space="preserve">
+    <value>Anmeldedaten</value>
+  </data>
+  <data name="strButtonApply" xml:space="preserve">
+    <value>Anwenden</value>
+  </data>
+  <data name="strOptionsThemeChangeWarning" xml:space="preserve">
+    <value>Warnung: Ein Neustart ist erforderlich, um Änderungen am Theme anzuwenden.</value>
+  </data>
+  <data name="strPropertyNameRedirectClipboard" xml:space="preserve">
+    <value>Zwischenablage</value>
+  </data>
+  <data name="strPropertyDescriptionRedirecClipboard" xml:space="preserve">
```

Comment:
`strPropertyDescriptionRedirecClipboard`

typo ... und würde die Beschreibung in `Umleitung der Zwischenablage erlauben` ändern

Reply:
erledigt.